.gitignore

```diff
@@ -13,4 +13,5 @@ web/app/yarn-error.log
 .protoc
 .gorun
 .dep*
+**.gogen*
```

Comment:
`go generate ./cli` generates the file `cli/static/generated_templates.gogen.go`. This will ignore that file if we are testing `go generate` in development.